src/figuresClasses.ts

```diff
@@ -1,19 +1,81 @@
 export interface Figure {
-
+  shape : string;
```

Comment:
Let's create the `Shape` type here with the appropriate values so that it could restrict the values that passed to the shape property

`shape: Shape`